Mathlib/MeasureTheory/Constructions/Pi.lean

```diff
@@ -743,12 +744,53 @@ theorem measurePreserving_piEquivPiSubtypeProd {ι : Type u} {α : ι → Type v
   exact Fintype.prod_subtype_mul_prod_subtype p fun i => μ i (s i)
 #align measure_theory.measure_preserving_pi_equiv_pi_subtype_prod MeasureTheory.measurePreserving_piEquivPiSubtypeProd
 
-theorem volume_preserving_piEquivPiSubtypeProd {ι : Type*} (α : ι → Type*) [Fintype ι]
+theorem volume_preserving_piEquivPiSubtypeProd (α : ι → Type*)
     [∀ i, MeasureSpace (α i)] [∀ i, SigmaFinite (volume : Measure (α i))] (p : ι → Prop)
     [DecidablePred p] : MeasurePreserving (MeasurableEquiv.piEquivPiSubtypeProd α p) :=
   measurePreserving_piEquivPiSubtypeProd (fun _ => volume) p
 #align measure_theory.volume_preserving_pi_equiv_pi_subtype_prod MeasureTheory.volume_preserving_piEquivPiSubtypeProd
 
+theorem measurePreserving_piCongrLeft (f : ι' ≃ ι) :
+    MeasurePreserving (MeasurableEquiv.piCongrLeft α f)
```

Comment:
Out of scope for this PR, since you're following the existing pattern, but... Shouldn't this be stated in the less-bundled way as:
```suggestion
    MeasurePreserving (Equiv.piCongrLeft α f)
```

Reply:
I'm not sure. I want to use both the `MeasurableEquiv` and `MeasurePreserving` parts at the same time when applying this theorem:
https://leanprover-community.github.io/mathlib4_docs/Mathlib/MeasureTheory/Integral/Lebesgue.html#MeasureTheory.MeasurePreserving.lintegral_map_equiv

That is easier to apply if this is formulated using `MeasurableEquiv`.